apps/paypal/views.py

```diff
@@ -161,6 +163,7 @@ def _paypal(request):
     return result
 
 
+@non_atomic_requests
```

Comment:
(and I'm not sure it's a view, it looks like it's called by the other one above)